common/clean_common.sh

```diff
@@ -390,7 +390,12 @@ function delete_cnis_bins_and_confs {
 function clear_mlnx_vfs {
     local mlnx_interfaces=$(get_auto_net_device 0)
 
+    local old_IFS=$IFS
```

Comment:
I don't understand this logic. IFS=' ' is assigned but never used

Reply:
IFS is a bash internal variable called the `internal field separator`, it controls what character are considered delimiters in for loops, (see https://bash.cyberciti.biz/guide/$IFS).
It seems the jenkins change its default value which cause the logic in this function to fail. To not be affected by the jenkins changes i am setting it here so that whatever its value in the environment, it would not affect the CI.